site/app/templates/autograding/AutoChecks.twig

```diff
@@ -64,6 +64,11 @@
                 <span onclick="openPopUp('{{ popup_css_file }}', '{{ element.title }}', {{ index }}, {{ check_index }}, {{ side }});" class="key_to_click" tabindex="0">
                     <i class="fas fa-window-restore" style="cursor: pointer;"></i>
                 </span>
+                {% if side == 0 %}
+                <span onclick="downloadTestCaseResult({{ index }}, '{{ check.filename }}', {{ display_version }}, '{{ gradeable_id }}', '{{ who }}')" tabindex="0">
```

Comment:
Should add a `title` value to this span to include some text on what this does for accessibility.